shape_completion_training/src/shape_completion_training/model/filepath_tools.py

```diff
@@ -1,12 +1,21 @@
-
 from __future__ import print_function
 import os
+import time
 from os.path import join
 import shutil
 import datetime
 import subprocess
 import json
 
+import git
+
+
+def unique_trial_name(*names):
+    now = str(int(time.time()))
```

Comment:
why not use datetime and make the string a bit more meaningful?
I thought that is what you do currently

Reply:
hm yea I wonder if I've copied this from the wrong place. I'll update with datetime